packages/back-end/src/services/python.ts

```diff
@@ -16,12 +16,44 @@ type PythonServerResponse<T> = {
   results: T;
 };
 
+function parseEnvInt(
```

Comment:
small issue: This will still return a value bigger or smaller than min/max if that's the `defaultValue`.

I guess it can also be the engineer responsibility to make sure that defaultValue falls within min & max.